package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "server": "webpack-dev-server --open",
+    "lint:js": "eslint",
+    "lint:css": "stylelint **/*.{css,scss}"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/simandebvu/todo-list.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/simandebvu/todo-list/issues"
+  },
+  "homepage": "https://github.com/simandebvu/todo-list#readme",
+  "devDependencies": {
```

Comment:
**REQUIRED.**
Let's also add `webpack-dev-server` dependency so that we can be able to start the server using `npm start` successfully.